validator/sawtooth_validator/journal/publisher.py

```diff
@@ -498,6 +498,16 @@ def queue_batch(self, batch):
         for observer in self._batch_observers:
             observer.notify_batch_pending(batch)
 
+    def can_accept_batch(self):
+        can_accept = len(self._pending_batches) < \
+            (2 * self._publish_count_average.value)
```

Comment:
Can you add a comment explaining the logic behind using this calculation to determine the high-water mark?